geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/MiniCluster.scala

```diff
@@ -52,15 +60,19 @@ case object MiniCluster extends LazyLogging {
       }
       connector.securityOperations().changeUserAuthorizations(name, auths)
     }
+
+    AccumuloVersion.close(connector)
+
+    logger.info("Started Accmulo minicluster")
```

Comment:
Shouldn't this be right after the cluster.start?

Reply:
i thought it made sense to add the user permissions stuff in the startup time, since the mini cluster isn't available until that returns, but we could log that separately if you want.

Reply:
Cool.  I didn't read closely enough.